job_application/tests.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from jobboard.models import Job
+from student.models import Student, EducationalInstitution
+from django.contrib.auth.models import User
+from job_application.models import Application
+import datetime
+from django.db.models import QuerySet
+
+
+# The tests use the data that is already in the db from the migration files
+@pytest.fixture
```

Comment:
You already have such fixture for institution_a for the students module (https://github.com/redhat-beyond/SJMaster/blob/main/student/tests.py#L10) , so instead of creating it here again (duplicated code), you should move it to `conftest.py`.
Same thing applies for all the other fixtures that are already created and used in more than one test file.